samples/highcharts/demo/combo-meteogram/demo.js

```diff
@@ -36,30 +29,177 @@ function Meteogram(xml, container) {
     this.pressures = [];
 
     // Initialize
-    this.xml = xml;
+    this.json = json;
     this.container = container;
 
     // Run
     this.parseYrData();
 }
 
-/**
- * Function to smooth the temperature line. The original data provides only whole degrees,
- * which makes the line graph look jagged. So we apply a running mean on it, but preserve
- * the unaltered value in the tooltip.
- */
-Meteogram.prototype.smoothLine = function (data) {
-    var i = data.length,
-        sum,
-        value;
-
-    while (i--) {
-        data[i].value = value = data[i].y; // preserve value for tooltip
-
-        // Set the smoothed value to the average of the closest points, but don't allow
-        // it to differ more than 0.5 degrees from the given value
-        sum = (data[i - 1] || data[i]).y + value + (data[i + 1] || data[i]).y;
-        data[i].y = Math.max(value - 0.5, Math.min(sum / 3, value + 0.5));
+Meteogram.dictionary = {
```

Comment:
Maybe it will be good to show where the dictionary comes [from](https://api.met.no/weatherapi/weathericon/2.0/documentation)?